### DISCUSSION TITLE
electron-vite vs vite-plugin-electron

### DISCUSSION THREAD

**Discussion by Oleksii14** [alex8088/electron-vite#147] [General]
Hey!
Could you provide some info of cases when we need to use vite-plugin-electron and simpler vite.config, and when to use electron-vite? What are the main differences and what tool to use in which case?

**Comment by alex8088** [comment]
electron-vite is a build tool which is very different from a plugin. It has a complete scaffolding tool, full documentation and supports more features(such as asset handling, source code protection, etc.). it aims to provide a faster and leaner DX for Electron. How to choose depends on your scene. electron-vite is a good option.
